exercises/run-length-encoding/src/example/java/RunLengthEncoding.java

```diff
@@ -0,0 +1,47 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RunLengthEncoding {
+    public String encode(String data) {
+        String encodedData = "";
+
+        for (int i = 0; i < data.length(); i++) {
+            char c = data.charAt(i);
+
+            if (i != 0 && c == data.charAt(i - 1)) continue;
```

Comment:
Yes, I agree, it's not so bad when it's on the same line as the header. But it's probably still a bit clearer to put it on it's own line with curly brackets. That way it's easier to see that it's an if statement :)